### DISCUSSION TITLE
Media filter is not working in my wincercms ver

### DISCUSSION THREAD

**Discussion by spanjaan** [wintercms/winter#590] [Q&A]
`<img src="{{ 'banner.jpg' | media }}" />`
this media filter is not working my project. anyone help me how to solve this problem? my wintercms version is 1.2
![Screenshot (9)](https://user-images.githubusercontent.com/64153858/177002587-08a7d9dd-ce7a-473e-a980-5b95ce19af96.png)
![Screenshot (10)](https://user-images.githubusercontent.com/64153858/177002594-fbb30dd6-ef78-4194-80b3-10eb6ee9e839.png)

**Comment by LukeTowers** [comment]
@Anjaan2077 are you using a public folder? What kind of server software are you using (nginx, apache, etc)? What is the configuration  that you are using for your Winter site?

**Comment by AIC-BV** [comment]
In your example code you search for banner.jpg but your image is called cover-image.jpg
